2024-04-29 08:54:38.166 +02:00 [INF] Starting up the service...
2024-04-29 08:54:39.311 +02:00 [ERR] Failed to generate and save report on attempt 1/5
Axpo.PowerServiceException: Error retrieving power volumes
   at Axpo.PowerService.CheckThrowError()
   at Axpo.PowerService.GetTradesAsync(DateTime date)
   at ReportGeneratorLogic.Services.PowerTradeService.GetTradesAsync(DateTime date) in C:\Repos\ReportGenerator\ReportGeneratorLogic\Services\PowerTradeService.cs:line 11
   at ReportGenerator.Program.GenerateAndSaveReport() in C:\Repos\ReportGenerator\ReportGeneratorApp\Program.cs:line 141
2024-04-29 08:55:37.808 +02:00 [INF] Report generated and saved to: ./reports/PowerPosition_20240430_202404290655.csv
2024-04-29 09:20:40.035 +02:00 [INF] Starting up the service...
2024-04-29 09:25:46.224 +02:00 [INF] Starting up the service...
2024-04-29 09:25:59.236 +02:00 [INF] Report generated and saved to: ./reports/PowerPosition_20240430_202404290725.csv
2024-04-29 09:38:21.507 +02:00 [INF] Starting up the service...
2024-04-29 09:38:31.072 +02:00 [INF] Report generated and saved to: ./reports/PowerPosition_20240430_202404290738.csv
2024-04-29 09:43:21.904 +02:00 [INF] Starting up the service...
2024-04-29 15:17:46.899 +02:00 [INF] Starting up the service...
2024-04-29 15:21:03.985 +02:00 [INF] Starting up the service...
2024-04-29 15:21:11.926 +02:00 [INF] Report generated and saved to: ./reports/PowerPosition_20240430_202404291321.csv
